apps/api-journeys/src/app/modules/userTeam/userTeam.resolver.ts

```diff
@@ -29,8 +29,17 @@ export class UserTeamResolver {
   @UseGuards(GqlAuthGuard, CaslGuard)
   async userTeams(
     @CaslAccessible('UserTeam') accessibleUserTeams: Prisma.UserTeamWhereInput,
-    @Args('teamId') teamId: string
+    @Args('teamId') teamId: string,
+    @Args('includeGuests') includeGuests: boolean
   ): Promise<UserTeam[]> {
+    if (!includeGuests) {
+      return await this.prismaService.userTeam.findMany({
+        where: {
+          role: { not: UserTeamRole.guest },
+          AND: [accessibleUserTeams, { teamId }]
+        }
```

Comment:
please do not have a where outside the array:

wrong:
```JS
        where: {
          role: { not: UserTeamRole.guest },
          AND: [accessibleUserTeams, { teamId }]
        }
```

right:
```JS
        where: {
          AND: [accessibleUserTeams, { teamId, role: { not: UserTeamRole.guest } }]
        }
```

Reply:
In any case this should now be:
```suggestion
        where: {
          AND: [accessibleUserTeams, { ...filter, teamId }]
        }
```